test/models/work_test.rb

```diff
@@ -0,0 +1,52 @@
+require "test_helper"
+
+describe Work do
+
+  it 'can be instantiated' do
+    (Work.new).must_be_kind_of Work
+  end
+
+  it "will have the required fields" do
+    work = works(:test_book)
+    [:category, :title, :creator, :publication_year, :description].each do |check|
+      expect(work).must_respond_to check
+    end
+  end
+
+  describe "relationships" do
+    # Your tests go here
+  end
+
+  describe "validations" do
+    let work = works(:test_book)
+    it 'must have a title' do
+      work.title = nil
+
+      expect(work.valid?).must_equal false
+      expect(work.errors.messages).must_include :vin
```

Comment:
I think you wanted to verify that the title wasn't valid. 

```suggestion
      expect(work.errors.messages.keys).must_include :title
```